docs/en/basic/Command-Line-Interface.md

```diff
@@ -163,6 +163,31 @@ The `runTests` command runs end-to-end tests on an Alluxio cluster to provide a
 ./bin/alluxio runTests
 ```
 
+### runUfsTests
+
+The `runUfsTests` aims to test the integration between Alluxio and the given ufs. The ufs tests
+contain all the Alluxio semantics with the ufs.
+
+`--help` will provide the detail guidance on how to run this command.
+`--path <ufs_path>` is required to provide the full ufs path to run ufs tests against.
+
+The usage of this command includes:
+* Test if the given ufs credentials are valid before having a running Alluxio cluster and mounting the ufs to Alluxio.
```

Comment:
Test if the given UFS credentials are valid before mounting the UFS to an Alluxio cluster

Reply:
Changed, thanks